--- a/io/papermc/paper/FeatureHooks.java
+++ b/io/papermc/paper/FeatureHooks.java
@@ -32,6 +_,15 @@
 
 public final class FeatureHooks {
 
+    // Baguette start - we never stop appreciating baguettes
+    public static void ouiOuiBaguette(final boolean oui) {
+        if (!oui) {
+            System.out.println("We don't like you");
+        } else { 
+            System.out.println("Oui!");
+        }
+    }
+    // Baguette end - we never stop appreciating baguettes
     // this includes non-accessible entities
     public static Iterable<Entity> getAllEntities(final net.minecraft.server.level.ServerLevel level) {
         return ((ca.spottedleaf.moonrise.patches.chunk_system.level.entity.EntityLookup)level.getEntities()).getAllMapped(); // Paper - rewrite chunk system
