--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -26,6 +_,12 @@
      */
     Key BRAND_CANVAS_ID = Key.key("canvasmc", "canvas");
     // Canvas end - Rebrand
+    // Baguette start - Rebrand
+    /**
+     * The brand id for Baguette.
+     */
+    Key BRAND_BAGUETTE_ID = Key.key("canvasmc", "baguette");
+    // Baguette end - Rebrand
 
     /**
      * Gets the {@code ServerBuildInfo}.
