From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Toffikk <slodkitofik@gmail.com>
Date: Thu, 24 Jul 2025 21:37:59 +0200
Subject: [PATCH] feature panic


diff --git a/net/minecraft/CrashReport.java b/net/minecraft/CrashReport.java
index 84247ddc9e063676c59a6e62da1eb919320bd18f..2ab6f09035a910c4f1c57b74818bba48102bc056 100644
--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -219,6 +219,6 @@ public class CrashReport {
 
     public static void preload() {
         //MemoryReserve.allocate(); // Paper - Disable memory reserve allocating
-        new CrashReport("Don't panic!", new Throwable()).getFriendlyReport(ReportType.CRASH);
+        new CrashReport("Do panic!", new Throwable()).getFriendlyReport(ReportType.CRASH); // Baguette - panic.
     }
 }
