From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Toffikk <slodkitofik@gmail.com>
Date: Thu, 24 Jul 2025 21:37:59 +0200
Subject: [PATCH] feature panic


diff --git a/net/minecraft/CrashReport.java b/net/minecraft/CrashReport.java
index effd79d743d17db2a9f852c8bf3c97666b9ac40f..4557e132677e00866997d136129c913b1bae5d94 100644
--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -216,6 +216,6 @@ public class CrashReport {
 
     public static void preload() {
         //MemoryReserve.allocate(); // Paper - Disable memory reserve allocating
-        new CrashReport("Don't panic!", new Throwable()).getFriendlyReport(ReportType.CRASH);
+        new CrashReport("Do panic!", new Throwable()).getFriendlyReport(ReportType.CRASH); // Baguette - panic.
     }
 }
