From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Toffikk <slodkitofik@gmail.com>
Date: Thu, 24 Jul 2025 21:37:59 +0200
Subject: [PATCH] feature panic


diff --git a/net/minecraft/CrashReport.java b/net/minecraft/CrashReport.java
index c2b6b0625095aa3645872eecbae4280bbadf85f2..5d63990a2f3cc81d21425b4e10f5cd0c54857d37 100644
--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -220,6 +220,6 @@ public class CrashReport {
 
     public static void preload() {
         //MemoryReserve.allocate(); // Paper - Disable memory reserve allocating
-        new CrashReport("Don't panic!", new Throwable()).getFriendlyReport(ReportType.CRASH);
+        new CrashReport("Do panic!", new Throwable()).getFriendlyReport(ReportType.CRASH); // Baguette - panic.
     }
 }
